package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "lb4-log-extension-example",
+  "version": "1.0.0",
+  "description": "An example extension project for LoopBack Next",
+  "main": "index.js",
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
+    "build": "npm run build:lib && npm run build:lib6",
+    "build:lib": "tsc --target es2017 --outDir dist",
+    "build:lib6": "tsc --target es2015 --outDir dist6",
+    "build:watch": "tsc --watch",
+    "clean": "rm -rf dist dist6",
+    "lint": "tslint -c tslint.full.json --project tsconfig.json --type-check",
+    "lint:fix": "npm run lint -- --fix",
+    "prepublish": "npm run build",
+    "pretest": "npm run clean && npm run build",
+    "test": "mocha",
+    "posttest": "npm run lint",
+    "test:dev": "mocha && npm run lint"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/strongloop/lb4-log-extension-example.git"
```

Comment:
Have we agreed on the naming convention for example repositories yet? Considering that all lb-next related repositories are using `loopback-next-` prefix so far, I am proposing to follow the same convention for consistency, until https://github.com/strongloop/loopback-next/issues/614 is resolved.